targets/unity

```diff
@@ -17,9 +17,15 @@ CHROOTETC='unity-autostart.desktop unity-profiled'
 . "${TARGETSDIR:="$PWD"}/common"
 
 ### Append to prepare.sh:
-install unity unity-2d ubuntu-artwork gnome-session nautilus \
-        ubuntu-settings,ubuntu~precise= \
-        ubuntu-session,ubuntu~precise+ubuntu~quantal+ubuntu~raring+ubuntu~saucy= \
+# Note that whitespace is important for syntax...
+# See distropkgs method in installer/prepare.sh
+install unity ubuntu-artwork gnome-session nautilus \
+        `# unity-2d only exists on ubuntu trusty and precise.`\
+        ubuntu~trusty+ubuntu~precise=unity-2d, \
+        `# ubuntu-settings does not exist on ubuntu precise.`\
+        ubuntu~precise=,ubuntu-settings \
+        `# ubuntu-session does not exist on the following releases.`\
+      ubuntu~precise+ubuntu~quantal+ubuntu~raring+ubuntu~saucy=,ubuntu-session \
```

Comment:
Please do not interleave code and comments.

Also, avoid comments such as `ubuntu-settings does not exist on ubuntu precise.` which basically repeat what the code does just below.